### PR TITLE
Feat/document pb column

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/md/faq.md [hunk 1]
```diff
@@ -532,4 +533,90 @@ func main() {
...
+	}
+	if b, ok := src.([]byte); ok {
+		if err := proto.Unmarshal(b, x); err != nil {
+			return err
+		}
+		return nil
+	}
+	return fmt.Errorf("unexpected type %T", src)
+}
+
```
Comment (a8m) [inline doc/md/faq.md:579]:
```suggestion
```

File: doc/md/faq.md [hunk 2]
```diff
@@ -532,4 +533,90 @@ func main() {
...
+func (x *Hi) Scan(src interface{}) error {
+	if src == nil {
+		return nil
+	}
+	if b, ok := src.([]byte); ok {
+		if err := proto.Unmarshal(b, x); err != nil {
+			return err
+		}
+		return nil
```
Comment (a8m) [inline doc/md/faq.md:575]:
```suggestion
		return proto.Unmarshal(b, x)
```

File: doc/md/faq.md [hunk 3]
```diff
@@ -532,4 +533,90 @@ func main() {
...
+
+```
+
+We add a new `field.Bytes` to our schema, setting the generated protobuf struct as its underlying `GoType`:
+
+```go
+// Fields of the Message.
+func (Message) Fields() []ent.Field {
+	return []ent.Field{
+		field.Bytes("hi").GoType(&pb.Hi{}),
```
Comment (a8m) [inline doc/md/faq.md:588]:
```suggestion
		field.Bytes("hi").
			GoType(&pb.Hi{}),
```

**Review by a8m — APPROVED** [review]
Thanks for adding this @rotemtam. Looks good

Please accept these changes and I'll merge this.

File: doc/md/faq.md (hunk 2 above)
Comment (a8m) [reply doc/md/faq.md:575]: Not sure if it's preferred, because it's less explicit. You can choose to accept it or reject it - both are fine.

Comment (rotemtam) [reply doc/md/faq.md:575]: Interesting suggestion, but  I think I prefer the more explicit version

**Pull request merged**